apps/web/src/lib/events/index.ts

```diff
@@ -1,3 +1,5 @@
+'use client'
```

Comment:
Do we really need to add `use client` in all these files. As far as understand is only needed when the component is a descendant of a server component. Childs of the client component don't need the use client directive. Maybe I got it wrong